charts/cluster-scanner/templates/secret.yaml

```diff
@@ -1,3 +1,18 @@
+{{- if (include "cluster-scanner.accessKeySecret" .) }}
```

Comment:
Instead of having this huge `if` statement, wouldn't it be better to have it arount the `sysdig_access_key` key (https://github.com/sysdiglabs/charts/pull/1287/files#diff-20cae487430fbff57280c8c521a4ed366a365219b54761f1dab3165a8583ee5dR25)?

Reply:
I have changed it, let me know if there is anything else 💪